.github/dependabot.yml

```diff
@@ -11,7 +11,12 @@ updates:
     schedule:
       interval: daily
 
-  - package-ecosystem: "gradle"
-    directory: "/"
+  - package-ecosystem: gradle
+    directory: /hedera-dependency-versions
     schedule:
-      interval: "weekly"
\ No newline at end of file
+      interval: weekly
+
+  - package-ecosystem: gradle
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
With having all versions in the standard format in `hiero-dependency-versions`, Dependbot will start working on this Repo for Gradle-managed dependencies as it does on others repos.